Runtime/RosMsgs/Vision/vision_msgs.cs

```diff
@@ -0,0 +1,117 @@
+using std_msgs;
+using std_msgs.msg;
+using geometry_msgs.msg;
+using sensor_msgs.msg;
+
+namespace vision_msgs
+{
+    namespace msg
+    {
+        public class BoundingBox3D : IRosMsg
+        {
+            string IRosMsg.GetRosType()
+            {
+                return "vision_msgs.msg.BoundingBox3D";
+            }
+
+            /// <summary>
+            /// The 3D position and orientation of the bounding box center.
+            /// </summary>
+            public Pose center;
+
+            /// <summary>
+            /// The size of the bounding box, in meters, surrounding the object's center pose.
+            /// </summary>
+            public Vector3 size;
+        }
+
+        public class ObjectHypothesis : IRosMsg
+        {
+            string IRosMsg.GetRosType()
+            {
+                return "vision_msgs.msg.ObjectHypothesis";
+            }
+
+            /// <summary>
+            /// The unique string ID of the object class.
+            /// </summary>
+            public string class_id;
+
+            /// <summary>
+            /// The probability or confidence value of the detected object.
+            /// </summary>
+            public double score;
+        }
+
+
+        public class ObjectHypothesisWithPose : ObjectHypothesis, IRosMsg
+        {
+            string IRosMsg.GetRosType()
+            {
+                return "vision_msgs.msg.ObjectHypothesisWithPose";
+            }
+
+            /// <summary>
+            /// The 6D pose of the object hypothesis. This pose should be
+            /// defined as the pose of some fixed reference point on the object,
+            /// such as the geometric center of the bounding box or the center of mass.
+            /// </summary>
+            public PoseWithCovariance pose;
+        }

```

Comment:
for ros2 there are must be field ObjectHypothesis named `hypothesis`
for ros1 there are must be fields `id`, `score`